drivers/storage/portworx/deployment.go

```diff
@@ -337,6 +336,20 @@ func (p *portworx) GetStoragePodSpec(
 		}
 	}
 
+	if cluster.Spec.Telemetry != nil && cluster.Spec.Telemetry.Enabled {
+		telemetryContainer := t.telemetryContainer()
+		if telemetryContainer != nil {
+			if len(telemetryContainer.Image) == 0 {
```

Comment:
The CCM container hasn't been pushed to a public docker hub yet. Hence we don't have a default. So we need to provide an image in the spec